go/ql/test/library-tests/semmle/go/dataflow/VarArgsWithFunctionModels/Flows.ql

```diff
@@ -18,69 +16,15 @@ class SummaryModelTest extends DataFlow::FunctionModel {
     or
     this.hasQualifiedName("github.com/nonexistent/test", "FunctionWithVarArgsParameter") and
     (inp.isParameter(_) and outp.isResult())
-    // or
-    // this.hasQualifiedName("github.com/nonexistent/test", "FunctionWithSliceOfStructsParameter")) and
-    // (inp.isParameter(0) and outp.isResult)
-    // or
-    // this.hasQualifiedName("github.com/nonexistent/test", "FunctionWithVarArgsOfStructsParameter")) and
-    // (inp.isParameter(0) and outp.isResult)
+    or
+    this.hasQualifiedName("github.com/nonexistent/test", "FunctionWithSliceOfStructsParameter") and
+    (inp.isParameter(0) and outp.isResult())
+    or
+    this.hasQualifiedName("github.com/nonexistent/test", "FunctionWithVarArgsOfStructsParameter") and
+    (inp.isParameter(0) and outp.isResult())
```

Comment:
I added this comment in the test
```
	// The following tests all fail because FunctionModel doesn't interact with access paths
```
I believe that when I originally wrote the tests and realised this I just left the models commented out. This time I thought it was better to have the models there and the tests still failing. I don't know what's best. Maybe just removing those tests, and having a comment noting this particular shortcoming of `FunctionModel`?